compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/ServiceValidationTest.java

```diff
@@ -985,7 +985,7 @@ public void testAnonymousRecordsAsFieldType() {
 
         diagnostic = diagnosticIterator.next();
         message = getErrorMessage(CompilationDiagnostic.INVALID_ANONYMOUS_FIELD_TYPE,
-                                  "record {|string name; int age;|}", "Query.class.profile");
+                                  "record {|string name; int age;|}", "Query.'class.profile");
```

Comment:
This quote is invalid. Is there a place we missed to escape?

```suggestion
                                  "record {|string name; int age;|}", "Query.class.profile");
```

Reply:
@ThisaruGuruge, this is compile time error, as the resource path has the quote, it is now visible in errors also. Runtime behavior is not changed

Reply:
Had a discussion with @DimuthuMadushan .. will reevaluate